### PR TITLE
Define cryptocurrencies on IB as inverse instruments

### PR DESCRIPTION
**Pull request by benjaminsingleton**
# Pull Request

Addresses #1358. Crypto instruments on Interactive Brokers are [inverse](https://docs.nautilustrader.io/api_reference/model/instruments.html#nautilus_trader.model.instruments.base.Instrument.is_inverse) instruments. A `cashQty` must be specified in the `IBOrder`.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## How has this change been tested?

Ran simple strategy to buy every few seconds. No errors encountered.

### CODE REVIEW

**Comment by cjdsellers** [comment]
Thanks @benjaminsingleton

Looking good to me :pray: , did you intend on adding more to this PR?

**Comment by benjaminsingleton** [comment]
Still a WIP. These changes resolve the `You must set Cash Quantity for this order` error, but now I'm encountering a new error: `API does not support trading in fractions for CashQty orders.`. I've tried decimal, double, int data types for cashQty, and I'm still getting the same error, so I need to continue investigating.

**Comment by cjdsellers** [comment]
> Still a WIP. These changes resolve the `You must set Cash Quantity for this order` error, but now I'm encountering a new error: `API does not support trading in fractions for CashQty orders.`. I've tried decimal, double, int data types for cashQty, and I'm still getting the same error, so I need to continue investigating.

My first thought would be we're somehow sending a type which represents fractions, but then I see you tried `int`. It makes sense that if you expressed an inverse quantity as USD, then cents wouldn't be acceptable. 

Thanks for the update!

**Comment by benjaminsingleton** [comment]
Update -- I'm able to buy / sell crypto without making any changes to code. Turns out there are settings that were blocking API calls and unhelpful error messages. I'm now doing testing because there is some unusual behavior (e.g., fills not being properly recognized), but I expect to have this PR ready soon.

**Comment by benjaminsingleton** [comment]
This is ready for review.

File: nautilus_trader/adapters/interactive_brokers/execution.py [hunk 1]
```diff
@@ -687,7 +691,7 @@ def _on_account_summary(self, tag: str, value: str, currency: str):
 
         self._account_summary_loaded.set()
 
-    def _handle_order_event(
+    def _handle_order_event(  # noqa: C901
```
Comment (cjdsellers) [inline nautilus_trader/adapters/interactive_brokers/execution.py:694]: Thanks for including the code too :+1:

**Review by cjdsellers — APPROVED** [review]
Looks like a good solution, many thanks!

**Pull request merged**